data_store/db/entities.py

```diff
@@ -49,18 +49,26 @@ class Funding(BaseModel):
     project: Mapped["Project"] = sqla.orm.relationship(back_populates="funding_records")
 
     __table_args__ = (
+        # migration auto-generate does not pick up these constrainsts
+        # TODO: Create migration to apply these constraints
```

Comment:
Good spot - let's fix this ASAP, please make a ticket in the tech backlog 👀

Reply:
What makes me nervous if when we migrate these... we might already have "non-compliant" data.

Reply:
yes, that's why I say we should do (try) it sooner rather than later. Especially before the next rounds 🤞